### DISCUSSION TITLE
where should we place parameters provider when create_catalog?

### DISCUSSION THREAD

**Discussion by FANNG1** [apache/gravitino#281] [General]
`provider` is used to create specify catalog like hiveCatalog. now it's hiding in  `CatalogCreateRequest.properties` with key `provider`,  should we place it in `CatalogCreateRequest` as the first level property? 
```java
class CatalogCreateRequest
{
   String name;
   Catalog.Type type;
   String comment;
   Map<String, String> properties;
}

```
cc @jerryshao

**Comment by FANNG1** [comment]
`provider` is placed in `CatalogCreateRequest` in #447

**Comment by jerryshao** [comment]
It is solved.
